WordPress/src/test/java/org/wordpress/android/ui/bloganuary/BloganuaryNudgeAnalyticsTrackerTest.kt

```diff
@@ -58,4 +59,37 @@ class BloganuaryNudgeAnalyticsTrackerTest : BaseUnitTest() {
             )
         }
     }
+
+    @Test
+    fun `WHEN trackLearnMoreOverlayShown is called THEN analyticsTracker is called correctly`() {
+        listOf(true, false).forEach { isPromptsEnabled ->
+            tracker.trackLearnMoreOverlayShown(isPromptsEnabled)
+
+            verify(analyticsTracker).track(
+                Stat.BLOGANUARY_NUDGE_LEARN_MORE_MODAL_SHOWN,
+                mapOf("prompts_enabled" to isPromptsEnabled.toString())
+            )
+        }
+    }
+
+    @Test
+    fun `WHEN trackLearnMoreOverlayDismissed is called THEN analyticsTracker is called correctly`() {
+        tracker.trackLearnMoreOverlayDismissed()
+
+        verify(analyticsTracker).track(
+            Stat.BLOGANUARY_NUDGE_LEARN_MORE_MODAL_DISMISSED
+        )
+    }
+
+    @Test
+    fun `WHEN trackLearnMoreOverlayActionTapped is called THEN analyticsTracker is called correctly`() {
+        BloganuaryNudgeLearnMoreOverlayAction.entries.forEach {
```

Comment:
👍

Reply:
Based on your comment here (https://github.com/wordpress-mobile/WordPress-Android/pull/19704#discussion_r1411410090) I will actually change this test to test all possibilities manually and compare against hard-coded expected values so we get failures if the enum is changed by mistake.